Source/DiabloUI/diabloui.cpp

```diff
@@ -922,7 +916,8 @@ void Render(const UiEdit *uiEdit)
 {
 	DrawSelector(uiEdit->m_rect);
 
-	Rectangle rect { { uiEdit->m_rect.x + 43, uiEdit->m_rect.y + 1 }, { uiEdit->m_rect.w - 86, uiEdit->m_rect.h } };
+	// To simulate padding we inset the region used to draw text in an edit control
+	Rectangle rect = MakeRectangle(uiEdit->m_rect).inset({ 43, 1 });

```

Comment:
Note:
This changes the size height or?
Previous it wasn't reduced, now it is is reduced by 2.
I think this is a improvement, just wanted to mention it.

Reply:
Yep, that's the first point I was calling out :D
> UiEdit text drawing regions now have their vertical dimension shrunk by 2 pixels. I don't think this matters in practice as when text overflows the clip region bounds it doesn't actually get clipped.

Reply:
Sorry my bad.